src/tests/functional/shared_test_classes/src/single_op/istft.cpp

```diff
@@ -0,0 +1,203 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/istft.hpp"
+
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "shared_test_classes/base/utils/ranges.hpp"
+
+namespace ov {
+namespace test {
+
+std::string ISTFTLayerTest::getTestCaseName(const testing::TestParamInfo<ISTFTParams>& obj) {
+    std::ostringstream result;
+
+    const auto& [data_shapes,
+                 frame_size,
+                 frame_step,
+                 signal_len,
+                 center,
+                 normalized,
+                 data_type,
+                 step_size_type,
+                 param_type,
+                 dev] = obj.param;
+
+    for (size_t s = 0lu; s < 2; s++) {
+        const auto& shape_item = data_shapes[s];
+        result << "IS" << s << "=(";
+        result << shape_item.first;
+        result << ")_TS=";
+
+        for (size_t i = 0lu; i < shape_item.second.size(); i++) {
+            result << "{";
+            result << ov::test::utils::vec2str(shape_item.second[i]);
+            result << "}_";
+        }
+    }
+
+    result << "FrameSize=" << frame_size << "_";
+    result << "StepSize=" << frame_step << "_";
+    result << "SignalLen=" << signal_len << "_";
+    result << "Center=" << center << "_";
+    result << "Normalized=" << normalized << "_";
+    result << "ModelType=" << data_type << "_";
+    result << "StepSizeType=" << step_size_type << "_";
+    result << "IsParameterOnly=" << param_type << "_";
+
+    result << "Device=" << dev;
+    return result.str();
+}
+
+void ISTFTLayerTest::SetUp() {
+    std::vector<InputShape> data_shapes;
+    int64_t frame_size;          // frame size value
+    int64_t frame_step;          // frame step value
+    int64_t signal_len;          // signal length value
+    bool center;                 // center
+    bool normalized;             // normalized
+    ElementType data_type;       // data_type
+    ElementType step_size_type;  // size/step type
+    utils::InputLayerType param_type;
+
+    std::tie(data_shapes,
```

Comment:
Can these vars be initialized like in the `getTestCaseName`?

Reply:
Actually yes, applied.